### ISSUE TITLE
Something's wrong with this...

### ISSUE THREAD

**Issue by Portalmaker233** [MC-U-Team/Music-Player#666]
Recently,I have encountered another issue about the mod launching problem.No idea about it,it just happened all of a sudden.
Could you guys please help me tackle it?I love this mod and I cannot get used to the game without it :(
Here it is.The file,which says a serious bug,should be reported to you. :p
I tested it and tried putting only music player and u teamcore in it,but failed.
[Output before game crash.txt](https://github.com/MC-U-Team/Music-Player/files/11412647/Output.before.game.crash.txt)

**Comment by HyCraftHD** [comment]
Hello, the issue is, that musicplayer cannot connect to twitch.tv.
Is it blocked?

**Comment by Portalmaker233** [comment]
Yes,it is.Maybe I should login the twitch and go for a setting?

**Comment by Portalmaker233** [comment]
Yes,it is.What should I do now?

**Comment by HyCraftHD** [comment]
Hm, that is a problem, note sure what to do to fix it (other than unblocking it).
I will try to find a solution
